example/nixos-flake/luks-lvm/configuration.nix

```diff
@@ -0,0 +1,15 @@
+{ config, pkgs, ... }:
+
+{
+  # point root to the physical partition that your LUKS encrypted LVM volume is on
+  boot.initrd.luks.devices = {
```

Comment:
why is this needed? disko should do this for you

Reply:
thanks! I didn't know this